Premier league.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>Whenever you find yourself using <code>apply</code>  - ask yourself whether there is a simpler way to do it !</p><p>Please simplify -</p><p>either use <code>drop_duplicates(['Team', 'Season'])</code>  or </p><p>use <code>groupby('Team')</code>  and <code>nunique</code></p><p> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>